medusa/common.py

```diff
@@ -584,6 +584,23 @@ def statusFromName(name, anime=False):
     }
 
     @staticmethod
+    def is_wanted_quality(status, show_obj):
+        """Return true if that quality is wanted based on show quality and episode status."""
+        cur_status, cur_quality = Quality.splitCompositeStatus(int(status) or -1)
+        allowed_qualities, preferred_qualities = show_obj.current_qualities
+
+        if cur_status not in {WANTED, DOWNLOADED, SNATCHED, SNATCHED_PROPER}:
```

Comment:
You can change it to tuple. That will not change the logic, it will have the exact same behaviour